presto-spi/src/main/java/com/facebook/presto/spi/ColumnHandle.java

```diff
@@ -13,6 +13,17 @@
  */
 package com.facebook.presto.spi;
 
+import static com.facebook.presto.spi.StandardErrorCode.NOT_SUPPORTED;
+
 public interface ColumnHandle
 {
+    default ColumnIdentity getColumnIdentity()
```

Comment:
Thanks @martint! How about putting the getColumn/TableIdentity and deserialized methods in ConnectorMetadata? i.e:
default ColumnIdentity getColumnIdentity(ConnectorSession session, ColumnHandle columnHandle) { // throw NOT_SUPPORTED exception }
default ColumnIdentity getTableIdentity(ConnectorSession session, CoHandle columnHandle) { // throw NOT_SUPPORTED exception }
default ColumnIdentity deserializeColumnIdentity(byte[] bytes) {  //throw NOT_SUPPORTED exception}
default TableIdentity deserializeTableIdentity(byte[] bytes) {  //throw NOT_SUPPORTED exception}